src/main.js

```diff
@@ -574,9 +577,21 @@ scene("game", () => {
 
 scene("over", () => {
     add([
-        text("Game over"),
-        pos(10, 10),
+        sprite("game-over"),
+        scale(2.5),
+        pos(30, 0)
     ])
+    add([
+        text("High Score: " + scoreText),
+        pos(gameWidth/2-130, gameHeight/2+90),
+    ])
+
+    add([
+        text("Click To Play Again"),
+        pos(gameWidth/2-160, gameHeight/2+170),
```

Comment:
And same here:

pos(gameWidth / 2, gameHeight / 2 + 170),
anchor("center"),

Reply:
Changes made as per Rowan, please test after it is merged into the main branch.